src/widgets/clear-all/clear-all.js

```diff
@@ -90,6 +95,12 @@ function clearAll({
         />,
         containerNode
       );
+    },
+
+    clearAll() {
+      if (this.clearAttributes.length > 0) {
```

Comment:
Why should we change the behaviour? Previously it could have been called and trigger a new search, but the widget sets autohideContainer to true by default, which removes the ability to call `clearRefinementsAndSearch`.